tools/port-drayage-webservice/src/main/java/com/leidos/inspection/InspectionActions.java

```diff
@@ -202,4 +202,20 @@ public void clear() {
         currentInspection = null;
     }
 
+    /**
+     * Method to return the number of waiting actions for UI tab notification
+     */
+    public int waitingActions() {
```

Comment:
In the future, when methods/functions are exactly the same or acts the same across, please consider moving them to a common file or utility class, so that there's no duplication in code and there's only one method to update in the future. For now, you can leave this as is.

Reply:
@maefromm Sure I could have probably made an interface that all these Action classes inherit from. Will keep that in mind of future improvement.